src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -501,6 +501,20 @@ class ZStackAdapter extends Adapter {
             } catch (error) {
                 debug('Response timeout (%s:%d,%d)', ieeeAddr, networkAddress, responseAttempt);
                 if (responseAttempt < 1 && !disableRecovery) {
+                    // No response could be because the radio of the end device is turned off:
+                    // Sometimes the coordinator does not properly set the PENDING flag.
+                    // Try to rewrite the device entry in the association table, this fixes it sometimes.
+                    const match =  await this.znp.request(
+                        Subsystem.UTIL, 'assocGetWithAddress',{extaddr: ieeeAddr, nwkaddr: networkAddress}
+                    );
+                    debug(`Node relation (${ieeeAddr}): ${match.payload.noderelation}`);
+                    if (match.payload.noderelation == 1) { // CHILD_RFD only
```

Comment:
Please also check if `supportsAssocRemove()` and ` supportsAssocAdd` are true. This is a custom command I added to the zstack firmware, not all my zstack firmware release support this command.

Reply:
fixed, thanks